easybuild/easyconfigs/j/JUBE/JUBE-2.4.2.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "VersionIndependentPythonPackage"
+
+name = "JUBE"
+version = "2.4.2"
+
+homepage = "https://www.fz-juelich.de/jsc/jube"
+description = """The JUBE benchmarking environment provides a script based
+framework to easily create benchmark sets, run those sets on different
+computer systems and evaluate the results.
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://apps.fz-juelich.de/jsc/jube/jube2/download.php?file=']
+sources = [SOURCE_TAR_GZ]
+checksums = ['d1de15e9792802f83521b582d1d144ec81e3d5a28c01dbd945288ea29b946729']
+
+options = {'modulename': 'jube2'}
+
+sanity_check_paths = {
+    'files': ['bin/jube'],
+    'dirs': [],
+}
+
```

Comment:
@SebastianAchilles looks like it should be possible to add a simple `sanity_check_command`, `jube --version`?

Reply:
Simple fix, so I've added this.